src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: $card-width;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: $header-text-color;
+  font-size: $font-basic-size;
+  line-height: 14px;
+
+  &__image {
+    display: flex;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__tittle {
+    text-align: left;
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: $font-basic-size;
+    font-weight: $roboto-medium;
+  }
+
+  &__review,
+  &__code,
+  &__price {
+    font-weight: 400;
+  }
+
+  &__review,
+  &__code {
+    font-size: 10px;
+  }
+
+  &__code,
+  &__price {
+    color: $card-price-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: $roboto-bold;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 0;
+    font-size: 14px;
+    text-align: center;
+    background: $v1-button-color;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    text-transform: uppercase;
+    font-weight: $roboto-bold;
```

Comment:
how does font-weight relate to Roboto font?
as for me, u can name it more abstractive

```SCSS
$font-weight_regular: 400;
...
$font-weight_bold: 700;
...
```